packages/workbox-core/test/bundle/node/controllers/test-WorkboxCore.js

```diff
@@ -15,6 +15,12 @@ describe(`WorkboxCore - ${process.env.NODE_ENV}`, function() {
       expect(core.INTERNAL).to.exist;
     });
 
+    it(`should return the same INTERNAL instance after first call`, function() {
+      const core = new WorkboxCore();
+      const internal = core.INTERNAL;
+      expect(core.INTERNAL).to.equal(internal);
```

Comment:
In this case I actually wanted it to check it was the same variable, but that being said the test isn't actually useful anymore so I'll remove it.